clients/java/src/main/java/io/zeebe/client/impl/response/BrokerInfoImpl.java

```diff
@@ -58,7 +59,7 @@ public int getPort() {
 
   @Override
   public String getAddress() {
-    return String.format("%s:%d", host, port);
+    return new InetSocketAddress(host, port).toString();
```

Comment:
Oops, this is not working as I expected 😅. I will fix that.

Reply:
I guess we can't reuse SocketUtil here since the client is on Java 8. I'm fine with copying it to the client code as an internal class, though, if it's small enough.